cspell.json

```diff
@@ -79,6 +79,7 @@
     "erc",
     "falsey",
     "fargate",
+    "Fieldeable",
```

Comment:
Where did you end up using it?

Reply:
The type exists already and it has a squiggly line :grimacing:  https://github.com/AztecProtocol/aztec-packages/blob/dfffe5d879df3131769e7e8709a69ed0a6c63b2e/yarn-project/foundation/src/serialize/serialize.ts#L124